### DISCUSSION TITLE
HBO/Max and Hulu buttons don't work for Chromecast

### DISCUSSION THREAD

**Discussion by leedomike** [Nerwyn/universal-remote-card#65] [Q&A]
Hi, I'm trying to set up my remote for buttons for HBO/Max and Hulu. The buttons don't do anything. I tried a custom action with HBO deeplinks (none available for Hulu) to no avail. Any help would be appreciated.

**Comment by Nerwyn** [comment]
[Max and Hulu are both broken](https://github.com/Nerwyn/android-tv-card/blob/main/src/models/maps/defaultSources.ts#L32-L39). You should also ask and look through the [Android TV deep linking guide on the HA forums](https://community.home-assistant.io/t/android-tv-remote-app-links-deep-linking-guide/567921). If you do figure it out, let me know!
